CMakeLists.txt

```diff
@@ -46,6 +77,7 @@ if(WIN32 AND MSVC_VERSION LESS 1900)
   message(FATAL_ERROR "Building with Microsoft compiler needs Latest Visual Studio 2015 (Community or better)")
 endif()
 
+option(ENABLE_MASON "Use mason for dependencies" OFF)
```

Comment:
A warning feels right to me, since they don't conflict on all platforms and I hope I can find a way to ensure they work on all platforms eventually. I will plan to add a warning with a link to a ticket on the issue. Sound good @TheMarex?